dokku

```diff
@@ -195,7 +195,7 @@ case "$1" in
     done < "$DOKKU_SCALE_FILE"
 
     dokku_log_info1 "Running post-deploy"
-    plugn trigger post-deploy $APP $port $ipaddr
+    plugn trigger post-deploy $APP $port $ipaddr $IMAGE_TAG
```

Comment:
The `pre-deploy` hook is:

```shell
plugn trigger pre-deploy $APP $IMAGE_TAG
```

Maybe we should follow that and instead do:

```shell
plugn trigger pre-deploy $APP $IMAGE_TAG $ipaddr $port
```

Or similar? What uses the port/ipaddr from here that cannot retrieve it from some other helper?